Sources/MapboxMaps/Location/Pucks/Puck2D.swift

```diff
@@ -17,14 +17,19 @@ public struct Puck2DConfiguration: Equatable {
     /// The size of the images, as a scale factor applied to the size of the specified image.
     public var scale: Value<Double>?
 
+    /// Flag determining if the horizontal accuracy ring should be shown arround the Puck. default value is false
+    public var showAccuracyRing: Bool
+
     public init(topImage: UIImage? = nil,
                 bearingImage: UIImage? = nil,
                 shadowImage: UIImage? = nil,
-                scale: Value<Double>? = nil) {
+                scale: Value<Double>? = nil,
+                showAccuracyRing: Bool = false) {
```

Comment:
How did we decide what to set as the default?

Reply:
currently this feature(accuracy ring) is not supported on android, and all requests from customers have been to disable this accuracy ring. It feels like the more natural direction